lib/std/jsfetch.nim

```diff
@@ -72,7 +72,7 @@ func clone*(self: Response or Request): Response {.importjs: "#.$1()".}
 proc text*(self: Body or Request): Future[cstring] {.importjs: "#.$1()".}
```

Comment:
should that be `Response`?
https://developer.mozilla.org/en-US/docs/Web/API/Body/text

> The text() method of the Body mixin takes a Response stream and reads it to completion.

please check the other procs too (json was incorrect too but was now fixed)